src/reducers/reducer-forecast.js

```diff
@@ -0,0 +1,41 @@
+import { FETCH_FORECAST} from "../actions";
+
+
+const DEFAULT_STATE = {
+  forecast_data: {},
```

Comment:
Given that you're looking to display a list of cities, I'd recommend `forecast_data` being an array (and being camel cased :))